packages/foreman/foreman/foreman.spec

```diff
@@ -310,8 +306,7 @@ BuildRequires: %{?scl_prefix}npm(react-intl) < 3.0.0
 # end package.json dependencies BuildRequires
 
 # start specfile assets BuildRequires
-BuildRequires: %{?scl_prefix_ror}rubygem(jquery-turbolinks) >= 2.1
-BuildRequires: %{?scl_prefix_ror}rubygem(jquery-turbolinks) < 3.0
+BuildRequires: %{?scl_prefix_ror}rubygem(coffee-rails) >= 4.2.2
```

Comment:
You're missing the upper bounds. https://github.com/ekohl/theforeman-plumbing/blob/master/fm-packaging-update-deps would have done all of this for you and made sure that in the future it'd also remain consistently sorted.